src/components/DownloadLogsFilePopover.vue

```diff
@@ -0,0 +1,85 @@
+<template>
+  <ion-content>
+    <ion-list>
+      <ion-list-header>{{ translate('Log Id') }}</ion-list-header>
+      <ion-item button @click="downloadLogFile('logFile')">
+        {{ translate('Log file') }}
+      </ion-item>
+      <ion-item button :lines="log?.errorRecordContentId ? undefined : 'none'" @click="downloadLogFile('uploadedFle')">
```

Comment:
I think we should show error item disabled instead of hiding, if there is not error content.